fontforge/python.c

```diff
@@ -11149,6 +11075,181 @@ static struct MATH *SFGetMathTable(SplineFont *sf) {
 return(sf->MATH);
 }
 
+static void PyFFMathDeviceTable_dealloc(PyFF_MathDeviceTable *self) {
+    self->sf = NULL;
+    PyObject_Del(self);
+}
+
+static Py_ssize_t PyFF_MathDeviceLength( PyFF_MathDeviceTable *self ) {
+    struct MATH *math = SFGetMathTable(self->sf);
+    DeviceTable *devtab = *((DeviceTable **) (((char *) (math)) + self->devtab_offset ));
+    uint16_t px, low, high;
+    int count = 0;
+
+    if ( devtab==NULL ) return count;
+
+    low = devtab->first_pixel_size;
+    high = devtab->last_pixel_size;
+
+    for ( px=low; px<=high; ++px ) {
+        if ( devtab->corrections[px-low]!=0 ) {
+            count++;
+        }
+    }
+
+    return count;
+}
+
+static PyObject *PyFF_MathDeviceIndex( PyFF_MathDeviceTable *self, PyObject *index ) {
+    struct MATH *math = SFGetMathTable(self->sf);
+    DeviceTable *devtab = *((DeviceTable **) (((char *) (math)) + self->devtab_offset ));
+    uint16_t px, low, high;
```

Comment:
`low` and `high` are unused
```suggestion
    uint16_t px;
```